### PR TITLE
Students changed input styles + reset filter + student info new style.

### PR DESCRIPTION
**Pull request by MichaelValkiv**

### CODE REVIEW

File: src/app/admin-panel/admin-panel/students/students.component.ts [hunk 1]
```diff
@@ -66,8 +66,18 @@ export class StudentsComponent implements OnInit {
   }
 
   createStudent() {
-    this.newStudent.classId = this.selectedClassId;
-    // this.newStudent = new Student();
+    this.newStudent = new Student(
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.ts]: define default values in models constructor

File: src/app/admin-panel/admin-panel/students/students.component.ts [hunk 2]
```diff
@@ -66,8 +66,18 @@ export class StudentsComponent implements OnInit {
   }
 
   createStudent() {
-    this.newStudent.classId = this.selectedClassId;
-    // this.newStudent = new Student();
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.ts]: comment

File: src/app/admin-panel/admin-panel/students/students.component.ts (hunk 1 above)
Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.ts]: We must do this for adding class id.

File: src/app/admin-panel/admin-panel/students/students.component.ts (hunk 2 above)
Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.ts]: Deleted.

File: src/app/admin-panel/admin-panel/students/students.component.html [hunk 3]
```diff
@@ -36,78 +36,83 @@
   </ng-template>
   <ng-template pTemplate="summary">
       <div style="text-align:left">
-    <button type="button" (click)="createStudent()" pButton icon="pi pi-plus" class="ui-button-raised" label="Додати учня"></button>
+        <button type="button" (click)="createStudent(); this.firstname.reset(); this.lastname.reset(); this.patronymic.reset(); this.dateOfBirth.reset()" pButton icon="pi pi-plus" class="ui-button-raised" label="Додати учня"></button>
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.html]: put all code into one handler

**Pull request merged**